util/ksonnet/ksonnet.go

```diff
@@ -158,18 +162,39 @@ func (k *ksonnetApp) ListParams() ([]*v1alpha1.KsonnetParameter, error) {
 	if err := json.Unmarshal([]byte(out), &ksParams); err != nil {
 		return nil, err
 	}
-	var params []*v1alpha1.KsonnetParameter
+	params := make(map[string]*v1alpha1.KsonnetParameter)
 	for _, ksParam := range ksParams.Data {
 		value := strings.Trim(ksParam.Value, `'"`)
-		params = append(params, &v1alpha1.KsonnetParameter{
+		params[fmt.Sprintf("%s/%s", ksParam.Component, ksParam.Key)] = &v1alpha1.KsonnetParameter{
```

Comment:
Minor, but I feel like we should have `KsonnetParameter.String()`  or similar.

Reply:
KsonnetParameter includes Value in addition to Component and Key. I would prefer to keep it as is.